rules/CodeQuality/Rector/Class_/ControllerMethodInjectionToConstructorRector.php

```diff
@@ -149,7 +149,15 @@ public function refactor(Node $node): ?Node
                 // skip allowed known objects
                 if ($this->isNames(
                     $param->type,
-                    [SymfonyClass::USER_INTERFACE, SymfonyClass::REQUEST, FosClass::PARAM_FETCHER, SymfonyClass::UUID, ...$entityClasses]
+                    [
+                        SymfonyClass::USER_INTERFACE,
+                        SymfonyClass::REQUEST,
+                        FosClass::PARAM_FETCHER,
+                        SymfonyClass::UUID,
+                        \Throwable::class,
+                        \Exception::class,
```

Comment:
Native class only not-scoped when it exists by downgrade php runner, but when we upgrade php version and native class not yet or no longer exists, it will be prefixed.

I think this should be on Enum class definition, or it maybe broke in the future scoped build.

Reply:
Native classes are excluded from scoping: https://github.com/rectorphp/rector/blob/main/vendor/rector/rector-symfony/rules/CodeQuality/Rector/Class_/ControllerMethodInjectionToConstructorRector.php#L153

Reply:
Still, this one is not preffixed. Not needed to extract.